pkg/util/validation/stack_test.go

```diff
@@ -9,6 +9,43 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestValidateStackName(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		title     string
+		stackName string
+		error     string
+	}{
+		{"empty", "", "a stack name may not be empty"},
+		{"valid", "foo", ""},
+		{
+			"slash", "foo/bar",
+			"a stack name may only contain alphanumeric, hyphens, " +
+				"underscores, or periods: invalid character '/'" +
```

Comment:
style nit: The multi-line cases would be more readable with named keys. Otherwise it takes a second when scanning these lines to see if one ends with `,` or `+`, which then decides whether it's a new field or a concatenation of the prior.

```go
{
  title: "slash",
  stackName: "foo/bar",
  error: ...
}
```